### PR TITLE
Add IPBX section

### PR DESCRIPTION
**Pull request by sboily**
Add freepbx web interface, elastix and xivo distro

### CODE REVIEW

**Comment by nodiscc** [comment]
Can web merge the SIP and IP PBX sections?

File: README.md [hunk 1]
```diff
@@ -270,11 +271,18 @@ _Mass mailing software - one message to many recipients_
...
   * [Kamailio](http://www.kamailio.org/w/) - Modular SIP server (registrar/proxy/router/etc) ([Source Code](https://github.com/kamailio/kamailio)) `GPLv2` `C`
   * [Ostel](https://dev.guardianproject.info/projects/ostel/wiki/Server_Documentation) - Secure SIP telephony setup with ZRTP encryption. `GPLv3` `Ruby`
 
+### IPBX
+
+ *[IPBX] (https://en.wikipedia.org/wiki/IP_PBX) telephony software*
+ 
+  * [Elastix](http://www.elastix.org) - Unified communications server software based on Asterisk [(Source Code](https://svn.code.sf.net/p/elastix/code/trunk/)) `GPLv2` `PHP`
+  * [Freepbx](http://www.freepbx.org) - Web-based open source GUI (graphical user interface) that controls and manages Asterisk [(Source Code](http://git.freepbx.org/projects/FREEPBX)) `GPLv2` `PHP`
+  * [XiVO](http://www.xivo.io) - Unified communications server software based on Asterisk ([Source Code](https://github.com/xivo-pbx)) `GPLv3` `python`
```
Comment (nodiscc) [inline README.md]: Pleas capitalize Python

**Comment by sboily** [comment]
If you prefer, yes we could but IPBX is not only SIP :) For example on XiVO the support is for SIP/SCCP devices and SIP/IAX trunk.

**Comment by nodiscc** [comment]
Ok to merge as-is. We'll find a way to merge these sections properly in the future.

**Comment by AndyR207** [comment]
Your brackets on the first two items in the IPBX section are wrong and you need to add full stops to the descriptions.

File: README.md [hunk 2]
```diff
@@ -270,11 +271,18 @@ _Mass mailing software - one message to many recipients_
...
 
-  * [Asterisk](http://www.asterisk.org/) - An easy to use but advanced IP PXB system, VoIP gateway and conference server. `GPLv2` `C`
+  * [Asterisk](http://www.asterisk.org/) - An easy to use but advanced IP PBX system, VoIP gateway and conference server. `GPLv2` `C`
   * [FreeSWITCH](https://freeswitch.org/) - Scalable open source cross-platform telephony platform. ([Source Code](https://freeswitch.org/stash/projects/FS/repos/freeswitch/browse)) `MPLv2` `C`
   * [Kamailio](http://www.kamailio.org/w/) - Modular SIP server (registrar/proxy/router/etc) ([Source Code](https://github.com/kamailio/kamailio)) `GPLv2` `C`
   * [Ostel](https://dev.guardianproject.info/projects/ostel/wiki/Server_Documentation) - Secure SIP telephony setup with ZRTP encryption. `GPLv3` `Ruby`
 
+### IPBX
+
+ *[IPBX] (https://en.wikipedia.org/wiki/IP_PBX) telephony software*
```
Comment (nodiscc) [inline README.md]: as mentioned by @AndyR207 please remove the extra space `] (`

File: README.md [hunk 3]
```diff
@@ -278,7 +278,7 @@ _Mass mailing software - one message to many recipients_
 
 ### IPBX
 
- *[IPBX] (https://en.wikipedia.org/wiki/IP_PBX) telephony software*
+ *[IPBX](https://en.wikipedia.org/wiki/IP_PBX) telephony software*
  
   * [Elastix](http://www.elastix.org) - Unified communications server software based on Asterisk [(Source Code](https://svn.code.sf.net/p/elastix/code/trunk/)) `GPLv2` `PHP`
```
Comment (AndyR207) [inline README.md]: [Elastix](http://www.elastix.org) - Unified communications server software based on Asterisk ([Source Code](https://svn.code.sf.net/p/elastix/code/trunk/)) `GPLv2` `PHP`

File: README.md [hunk 4]
```diff
@@ -278,7 +278,7 @@ _Mass mailing software - one message to many recipients_
...
   * [Freepbx](http://www.freepbx.org) - Web-based open source GUI (graphical user interface) that controls and manages Asterisk [(Source Code](http://git.freepbx.org/projects/FREEPBX)) `GPLv2` `PHP`
```
Comment (AndyR207) [inline README.md]: [Freepbx](http://www.freepbx.org) - Web-based open source GUI (graphical user interface) that controls and manages Asterisk. ([Source Code](http://git.freepbx.org/projects/FREEPBX)) `GPLv2` `PHP`

**Comment by AndyR207** [comment]
Still missing the full stops after the descriptions.

**Comment by sboily** [comment]
I'm sorry but i can't see where i missed something :(

File: README.md [hunk 5]
```diff
@@ -270,11 +271,18 @@ _Mass mailing software - one message to many recipients_
...
+  * [Asterisk](http://www.asterisk.org/) - An easy to use but advanced IP PBX system, VoIP gateway and conference server. `GPLv2` `C`
   * [FreeSWITCH](https://freeswitch.org/) - Scalable open source cross-platform telephony platform. ([Source Code](https://freeswitch.org/stash/projects/FS/repos/freeswitch/browse)) `MPLv2` `C`
   * [Kamailio](http://www.kamailio.org/w/) - Modular SIP server (registrar/proxy/router/etc) ([Source Code](https://github.com/kamailio/kamailio)) `GPLv2` `C`
   * [Ostel](https://dev.guardianproject.info/projects/ostel/wiki/Server_Documentation) - Secure SIP telephony setup with ZRTP encryption. `GPLv3` `Ruby`
 
+### IPBX
+
+ *[IPBX](https://en.wikipedia.org/wiki/IP_PBX) telephony software*
+ 
+  * [Elastix](http://www.elastix.org) - Unified communications server software based on Asterisk ([Source Code](https://svn.code.sf.net/p/elastix/code/trunk/)) `GPLv2` `PHP`
```
Comment (AndyR207) [inline README.md]: Should be

    [Elastix](http://www.elastix.org) - Unified communications server software based on Asterisk. ([Source Code](https://svn.code.sf.net/p/elastix/code/trunk/)) `GPLv2` `PHP`

File: README.md [hunk 6]
```diff
@@ -270,11 +271,18 @@ _Mass mailing software - one message to many recipients_
...
+  * [Freepbx](http://www.freepbx.org) - Web-based open source GUI (graphical user interface) that controls and manages Asterisk ([Source Code](http://git.freepbx.org/projects/FREEPBX)) `GPLv2` `PHP`
```
Comment (AndyR207) [inline README.md]: Should be

    [Freepbx](http://www.freepbx.org) - Web-based open source GUI (graphical user interface) that controls and manages Asterisk. ([Source Code](http://git.freepbx.org/projects/FREEPBX)) `GPLv2` `PHP`

File: README.md [hunk 7]
```diff
@@ -270,11 +271,18 @@ _Mass mailing software - one message to many recipients_
...
+  * [XiVO](http://www.xivo.io) - Unified communications server software based on Asterisk ([Source Code](https://github.com/xivo-pbx)) `GPLv3` `Python`
```
Comment (AndyR207) [inline README.md]: Should be

    [XiVO](http://www.xivo.io) - Unified communications server software based on Asterisk ([Source Code](https://github.com/xivo-pbx)) `GPLv3` `Python`

**Comment by nodiscc** [comment]
https://github.com/sboily/awesome-selfhosted/pull/1

@AndyR207 you can edit people's patches/PRs directly (`View files -> View (the whole file at...) -> Edit (pencil) -> Commit message -> [x] Submit as new pull request`), sometimes it is faster than reviewing here

**Comment by Kickball** [comment]
@nodiscc I have tried the method you outlined above with #536 and [#1](https://github.com/pierreozoux/awesome-selfhosted/pull/1). 

Is that the intended outcome?

**Comment by nodiscc** [comment]
Yes it is.
When https://github.com/pierreozoux/awesome-selfhosted/pull/1 is merged, the changes will be available in https://github.com/Kickball/awesome-selfhosted/pull/536.

**Pull request merged**